helm.tf

```diff
@@ -68,4 +68,9 @@ resource "helm_release" "this" {
       type  = try(set_sensitive.value.type, null)
     }
   }
+
+  depends_on = [
+    # Wait for EBS CSI, etc. to be installed first
```

Comment:
What if EBS CSI is not enabled?

Reply:
nothing in particular. If no addons are passed in via `eks_addons` then it will just treat it as a no-op. If any addons are passed in (kube-proxy, VPC CNI, etc.) then it will ensure those are provisioned before the `helm_releases` which should be what we want for 99% of the time.